setup/pkg/fieldlabs/environment_create.go

```diff
@@ -316,10 +319,16 @@ KUBECONFIG=/etc/kubernetes/admin.conf kubectl kots install %s-%s \
 				publicIPs["_"] = nil
 			}
 
+			proxy := false
+			if lab.Spec.Proxy {
+				proxy = true
+			}
+
 			lab.Status.InstanceToMake = Instance{
 				Name:        appLabSlug,
 				MachineType: "n1-standard-4",
 				BookDiskGB:  "200",
+				Proxy:       proxy,
```

Comment:
is there a reason not to just set the value of lab.Spec.Proxy here?

Reply:
My lack of go experience. What if in the `json`,  `lab.Spec.Proxy` is not provided? Is it going to be `nil`. In that case, what would the resulting `provisioner_pairs.json` be?
If not, and it would default to `false`, I'll change it.